lib/runner.js

```diff
@@ -910,6 +915,12 @@ Runner.prototype.run = function(fn) {
   this.on(constants.EVENT_RUN_END, function() {
     debug(constants.EVENT_RUN_END);
     process.removeListener('uncaughtException', uncaught);
+    process.on('uncaughtException', function(err) {
```

Comment:
This adding an `uncaughtException` handler every time the `end` event of a runner is called - this means for every finished runner, a new handler gets added. This, in turn, causes a `Possible EventEmitter memory leak detected. 11 uncaughtException listeners added to [process]. Use emitter.setMaxListeners() to increase limit` warning to be fired by node.

:(